content/metadata.yaml

```diff
@@ -18,6 +18,14 @@ authors:  #Below is a randomised order of authors. The author order will be deci
     funders:
       - Neurofibromatosis Therapeutic Acceleration Program
       - Children's Tumor Foundation
+  -
+    github: jaclyn-taroni
+    name: Jaclyn N Taroni [1]
```

Comment:
I do not feel that my contributions were equal to those of @jaybee84! I happy to go with the proposed ordering (e.g., dropping `[1]`) if that's a-okay with everyone else.

Reply:
I would like to re-propose equal authorship for you @jaclyn-taroni :)

Reply:
Okay, I will not die on this hill 😅

Reply:
:)